### ISSUE TITLE
¿Qué problemas surgen a traves de las HU para crear las entidades de la estructura de datos?

### ISSUE THREAD

**Issue by JoseJordanF** [JoseJordanF/Claqueta#62]
La estructura de datos viene definida por los problemas vistos en las HU, de esta manera que problemas son y qué entidades debemos definir y con que atributos

**Comment by JJ** [comment]
No uses issues genéricos, sino específicos. Por otro lado, todo issue se tiene que referir a una HU.

**Comment by JoseJordanF** [comment]
Hago issues específicos de los problemas extraídos de las HU

**Comment by JoseJordanF** [comment]
De esta manera? o directamente un issue directamente del problema que plantea?

**Comment by JJ** [comment]
Claro, el issue debe plantear un problema, no decir "vamos a resolver el problema aquí". ¿Qué es lo que quiere el aficionado y qué tiene que saber de él la lógica de negocio?

**Comment by JoseJordanF** [comment]
vale
